tsconfig.json

```diff
@@ -5,6 +5,29 @@
     "lib": ["es2020"],
     "module": "commonjs",
     "target": "es2020",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+
+    "strict": true,
+    "alwaysStrict": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true,
+    "strictPropertyInitialization": true,
+    "useUnknownInCatchVariables": true,
+
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "allowUnusedLabels": false,
+    "allowUnreachableCode": false,
+    "noImplicitOverride": true,
+    "noImplicitReturns": true,
+
+    //"noUncheckedIndexedAccess": true,
```

Comment:
we cant enable this yet because we use this syntax a lot:


```
error TS2488: Type '[PublicKey, number] | undefined' must have a '[Symbol.iterator]()' method that returns an iterator.

31     const [[userCollateralATA], [userRedeemableATA]] = findMultipleATAAddSync
```